### DISCUSSION TITLE
How to serve images for wordpress using imgproxy

### DISCUSSION THREAD

**Discussion by ghost** [imgproxy/imgproxy#753] [Q&A]
Hi i am new to this, i would like to know how to benefit from imgproxy in wordpress. My wordpress is hosted in one of the hosting services. What exactly i need to do to serve the images in imgproxy url?

**Comment by petrozavodsky** [comment]
Try using my WordPress plugin https://github.com/petrozavodsky/ImageProxy

**Comment by jarPotato** [comment]
This is what we ended up doing: https://github.com/jarPotato/imgproxy-wordpress
Apart from imgproxy it also has a few more features for overall html image optimisations - inspired by the optimisations next image applies.
